crates/nargo_cli/src/cli/execute_cmd.rs

```diff
@@ -122,45 +130,50 @@ fn extract_opcode_error_from_nargo_error(
 /// Resolve an [OpcodeLocation] using debug information generated during compilation
 /// to determine an opcode's call stack. Then report the error using the resolved
 /// call stack and any other relevant error information returned from the ACVM.
```

Comment:
This description seems outdated now that this takes OpcodeLocations, can you update?